package Simple_msgs.msgs;

message SimpleConnection
{
  required string parent      = 1;
  required string child = 2;
  optional string parent_port = 3;
  optional string child_port = 4;
}
